spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -483,13 +483,23 @@ public async Task RunWithTemporaryDatabaseAsync(Func<string, Task> testFunction,
     }
 
     public async Task RunWithTemporaryDatabaseAsync(string instanceId, string databaseId, Func<string, Task> testFunction, params string[] extraStatements)
+    => await RunWithTemporaryDatabaseAsync(instanceId, databaseId, testFunction, DatabaseDialect.GoogleStandardSql, extraStatements);
+
+    private async Task RunWithTemporaryDatabaseAsync(string instanceId, string databaseId, Func<string, Task> testFunction, DatabaseDialect databaseDialect, params string[] extraStatements)
     {
-        var operation = await DatabaseAdminClient.CreateDatabaseAsync(new CreateDatabaseRequest
+        var databaseCreateRequest = new CreateDatabaseRequest
```

Comment:
I would only add validatation like this, to protect ourselves in the future 

```suggestion
        if (databaseDialect == DatabaseDialect.PostgressSql && extraStatemes?.Length >= 1)
        {
            throw new ArgumentException("Postgres does not accept extra statements for DB creation").
        }
        var databaseCreateRequest = new CreateDatabaseRequest
```

Or what's better yet, but in another PR and as part of the work that @anuragsrivstv is doing on #2266, if it's Google SQL you use extra statements, else you execute those statements individually after database creation. But all of this in a separate PR for sure. Here, just the validation is enough.

Reply:
done will raise another PR for the same.